proto/def.bzl

```diff
@@ -183,9 +183,15 @@ go_proto_library = rule(
 # go_proto_library is a rule that takes a proto_library (in the proto
 # attribute) and produces a go library for it.
 
-def go_grpc_library(**kwargs):
-    # TODO: Deprecate once gazelle generates just go_proto_library
-    go_proto_library(compilers = [Label("//proto:go_grpc")], **kwargs)
+def go_grpc_library(name, **kwargs):
```

Comment:
Should this support passing in additional values for `compilers` for things like `@com_github_grpc_ecosystem_grpc_gateway_v2//protoc-gen-grpc-gateway:go_gen_grpc_gateway`?

Reply:
For custom scenarios I think you can always "drop down" to the lower-level `go_proto_library` and pass in whatever compilers you want. IMO this method is mostly a matter of convenience for the 90% of use-cases that want these two specific compilers.

Reply:
Ya totally fair, I only bring it up because the deprecation notice says `via go_grpc_library() rule` which didn't end up working for us since we use grpc-gateway, so maybe the deprecation message could mention both options to clear up any confusion

Reply:
That seems entirely reasonable